test/e2e/containers_conf_test.go

```diff
@@ -260,6 +260,7 @@ var _ = Describe("Verify podman containers.conf usage", func() {
 
 	It("using journald for container with container log_tag", func() {
 		SkipIfInContainer("journalctl inside a container doesn't work correctly")
+		SkipIfJournaldUnavailable()
```

Comment:
I'm not sure if my comment is the same as @Luap99's (below) because I can't tell if he's referring to just that one file which has a different setup.

This duplication bothers me. I would prefer to see only one `skip` here and in other places where there's `SkipIfInContainer("journald blah blah")`, and that `skip` should be your new one, `SkipIfJournaldUnavailable()`.

@sstosh do you have a way to figure out what `journalctl -n 1` does inside a container?
* if it *fails*, then, wonderful, no extra work
* if it *succeeds*, please add `SkipIfInContainer` or some other test-for-container *inside* `SkipIfJournaldUnavailable`, because it sounds like we can assume that journald will never be available inside a container

Either way, you can then completely replace `SkipIfInContainer` with `SkipIfJournaldUnavailable` in these select cases where the container skip is due to a need for journald.

Reply:
> @sstosh do you have a way to figure out what journalctl -n 1 does inside a container?

Even though journald is unavailable on the host machine,
`journalctl -n 1` returns `exit code 0` in the container.

```
$ journalctl -n 1
Hint: You are currently not seeing messages from other users and the system.
      Users in the 'systemd-journal' group can see all messages. Pass -q to
      turn off this notice.
No journal files were opened due to insufficient permissions.
$ echo $?
1

$ podman pull quay.io/libpod/fedora_podman
49ea75a29fef82d1e2ef4c3fe521ded2ecb1c9f21fc5e8d84cd08f1d6f6cca06

$ podman exec -it test /bin/bash

[root@f65522ab661e /]# journalctl -n 1
No journal files were found.
-- No entries --
[root@f65522ab661e /]# echo $?
0
```

Reply:
ok, thanks for checking